xbmc/interfaces/json-rpc/InputOperations.cpp

```diff
@@ -13,6 +13,8 @@
 #include "guilib/GUIKeyboardFactory.h"
 #include "guilib/GUIWindow.h"
 #include "guilib/GUIWindowManager.h"
+#include "input/ButtonTranslator.h"
+#include "input/InputManager.h"
```

Comment:
Is this include still used?

Reply:
No this is indeed unused and has been removed.